web/src/TabNav.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from "react"
+import { Link } from "react-router-dom"
+import { SidebarItem } from "./Sidebar"
+import { ResourceView } from "./HUD"
```

Comment:
the circular dependency between TabNav and HUD seems not good, maybe this should be moved to a common file?